packages/icrc-ledger-agent/CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
+### Changed
+- Use leb128 decoding for icrc3 certificates
```

Comment:
Should this be something like "Also support leb128 encoded last_block_hash icrc3 certificates", since the `icrc-ledger-agent` will now support both certificates?